lms/djangoapps/student_account/views.py

```diff
@@ -302,6 +311,45 @@ def _external_auth_intercept(request, mode):
         return external_auth_register(request)
 
 
+def _get_order_details(user):
+    """ Retrieve order details for the user. """
+    cache_key = CommerceConfiguration.CACHE_KEY + '.' + user.username
+    cached = cache.get(cache_key)
+    if cached:
+        return cached
+
+    order_details = []
+    allowed_certificates = ['professional', 'verified', 'credit']
+    try:
+        commerce_order_details = ecommerce_api_client(user).orders.get()
+    except Exception:  # pylint: disable=broad-except
```

Comment:
Remove this exception handling, instead check if the configuration is enabled then just fetch the data and let the exception get raised so that we can fix the actual root cause.
```python
if not api_config.enabled:
        log.warning('%s configuration is disabled.', api_config.API_NAME)
        return no_data
```